project/Dependencies.scala

```diff
@@ -125,6 +125,16 @@ object Dependencies {
     )
   }
 
+  val IbmEventStore = Seq(
+    libraryDependencies ++= Seq(
+      "org.apache.spark"    %% "spark-sql"        % "2.2.0",
+      "EventStore"          % "ibm-event-desktop" % "1.1.1",
```

Comment:
Licenses for these two

Reply:
Asked IBM, waiting.

Reply:
Eventstore license:
https://github.com/IBMProjectEventStore/EventStore-DeveloperPreview/blob/master/EventStore/License/LA_en

Apache spark license:
https://github.com/apache/spark/blob/master/LICENSE

What steps do we need to take to evaluate the IBM one ?

Reply:
Please add a comment noting the licenses like on other external deps